db/migrate/20230126135846_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] kindly create an `author_id` and a corresponding index. `t.references :author, foreign_key: { to_table: 'users' }`